org-roam-capture.el

```diff
@@ -715,7 +715,10 @@ the current value of `point'."
         (when (find-buffer-visiting new-file)
           (kill-buffer (find-buffer-visiting new-file)))
         (delete-file new-file))
-    (org-id-add-location (org-roam-capture--get :id) (org-roam-capture--get :new-file))
+    (when-let (file (or (org-roam-capture--get :new-file)
```

Comment:
you're missing at least a bracket here, so file is always `nil`, and same below.

Reply:
`when-let ` and `if-let` use the same SPEC, which "As a special case, interprets a SPEC of the form `(SYMBOL SOMETHING)` like `((SYMBOL SOMETHING))`", according to the documentation if `if-let`.